File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_LSTSUBREPORT.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_LSTSUBREPORT } from './NS_RP_LSTSUBREPORT.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_LSTSUBREPORTService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'REPORTCODE', 'IDX', 'REPORTFILENAME', 'PROCEDURENAME', 'TableMapping'];

    List: NS_RP_LSTSUBREPORT[] | undefined;
    ListFilter: NS_RP_LSTSUBREPORT[] | undefined;
    FormData!: NS_RP_LSTSUBREPORT;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_LSTSUBREPORT";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_LSTSUBREPORTService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_LSTSUBREPORTService : BaseService<NS_RP_LSTSUBREPORT, INS_RP_LSTSUBREPORTRepository>
    , INS_RP_LSTSUBREPORTService
    {
    private readonly INS_RP_LSTSUBREPORTRepository _NS_RP_LSTSUBREPORTRepository;
    public NS_RP_LSTSUBREPORTService(INS_RP_LSTSUBREPORTRepository NS_RP_LSTSUBREPORTRepository) : base(NS_RP_LSTSUBREPORTRepository)
    {
    _NS_RP_LSTSUBREPORTRepository = NS_RP_LSTSUBREPORTRepository;
    }
    }
    }

